presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcTableLayoutHandle.java

```diff
@@ -14,46 +14,42 @@
 package com.facebook.presto.plugin.jdbc;
 
 import com.facebook.presto.spi.ColumnHandle;
-import com.facebook.presto.spi.ConnectorPartition;
+import com.facebook.presto.spi.ConnectorTableLayoutHandle;
 import com.facebook.presto.spi.predicate.TupleDomain;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
-import static com.google.common.base.MoreObjects.toStringHelper;
 import static java.util.Objects.requireNonNull;
 
-public class JdbcPartition
-        implements ConnectorPartition
+public class JdbcTableLayoutHandle
+    implements ConnectorTableLayoutHandle
 {
-    private final JdbcTableHandle jdbcTableHandle;
-    private final TupleDomain<ColumnHandle> domain;
+    private final JdbcTableHandle table;
+    private final TupleDomain<ColumnHandle> tupleDomain;
 
-    public JdbcPartition(JdbcTableHandle jdbcTableHandle, TupleDomain<ColumnHandle> domain)
+    @JsonCreator
+    public JdbcTableLayoutHandle(@JsonProperty("table") JdbcTableHandle table,
```

Comment:
The code style is to either wrap none of the arguments or wrap all. We always wrap JSON constructors with more than one argument since the annotations are long and wrapping makes it easier to read:
```java
public JdbcTableLayoutHandle(
        @JsonProperty("table") JdbcTableHandle table,
        @JsonProperty("tupleDomain") TupleDomain<ColumnHandle> domain)